### PR TITLE
Ticket3

### PR DESCRIPTION
**Pull request by Stephen-I**

### CODE REVIEW

File: db/app.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const app = express();
+
+const { viewCategories, getMessage } = require("./controllers/app.controller");
+
+app.use(express.json());
+
+app.get("/api", getMessage);
+
+app.get("/api/categories", viewCategories);
+
```
Comment (Hard-Luck) [inline db/app.js:16]: We should have a app.use that catches all unmatching paths and returns a 404: "Not found" and a test for this

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      .then(({ body }) => {
+        expect(Array.isArray(body.categories)).toBe(true);
+      });
+  });
+  test("Categories should have properties of slug and description", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        body.categories.forEach((category) => {
```
Comment (Hard-Luck) [inline __tests__/app.test.js:33]: we can look at the categories in the test data and know what the length of the array should be, so I would test against that too

**Review by Hard-Luck — CHANGES_REQUESTED** [review]
Great solution we just need to add a length check to the tests because if the arry was empty we'd get false positives.

Also add a catching endpoint that returns 404 for unmatched paths.
